src/rules/dynamic-import-chunkname.js

```diff
@@ -0,0 +1,70 @@
+import docsUrl from '../docsUrl'
+
+module.exports = {
+  meta: {
+    docs: {
+      url: docsUrl('dynamic-import-chunkname'),
+    },
+    schema: [{
+      type: 'object',
+      properties: {
+        importFunctions: {
+          type: 'array',
+          items: {
+            type: 'string',
+          },
+        },
+        webpackChunknameFormat: {
+          type: 'string',
+        },
+      },
+    }],
+  },
+
+  create: function (context) {
+    const config = context.options[0]
+    const { importFunctions = [] } = config || {}
+    const { webpackChunknameFormat = '[0-9a-zA-Z-_/.]+' } = config || {}
+
+    const commentFormat = ` webpackChunkName: "${webpackChunknameFormat}" `
+    const commentRegex = new RegExp(commentFormat)
+
+    return {
+      [`CallExpression[callee.type="Import"],CallExpression[callee.name]`](node) {
+        const { callee: { name }} = node
+        if (name && !importFunctions.includes(name)) {
```

Comment:
i'm not sure if we can use this in node 4, which eslint supports. you may want to do `new Set(importFunctions)` and then use `.has`?